static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-country-name.js

```diff
@@ -24,7 +24,11 @@ export const countryName: EpicABTest = makeEpicABTest({
 
     canRun: () => {
         const geolocation = geolocationGetSync();
-        return geolocation !== 'US' && geolocation !== 'GB';
+        return (
+            geolocation !== 'US' &&
+            geolocation !== 'GB' &&
+            countryNames[geolocationGetSync()]
```

Comment:
Can you use the `const` created here?

```suggestion
            countryNames[geolocation]
```